system/services/forms/FormsService.cfc

```diff
@@ -1475,6 +1475,29 @@ component displayName="Forms service" {
 		return "dynamicform-" & LCase( Hash( SerializeJson( arguments.definition ) ) );
 	}
 
+	private string function _runDefaultValueFunction( required string objectName, required string default ) {
+		var defaultValue = arguments.default ?: "";
+
+		if ( Len( Trim( defaultValue ) ) ) {
+			switch( ListFirst( defaultValue, ":" ) ) {
+				case "cfml":
+					defaultValue = Evaluate( ListRest( defaultValue, ":" ) );
+				break;
+				case "closure":
+					var func = Evaluate( ListRest( defaultValue, ":" ) );
+					defaultValue = func( {} );
+				break;
+				case "method":
+					var obj = _getPresideObjectService().getObject( arguments.objectName );
+
+					defaultValue = obj[ ListRest( defaultValue, ":" ) ]( {} );
+				break;
+			}
+		}
+
+		return defaultValue;
```

Comment:
It could make sense here to do:

```
return defaultValue ?: "";
```

This way, if any existing implementations return `null` in some circumstances, they won't error and instead just have no default value which would be expected.

I'll make this adjustement while merging, but thought be good to mention here.